posix/clone

```diff
@@ -1,13 +1,14 @@
 #!/bin/sh
 
-mkdir -p ${DRONE_WORKSPACE}
+sudo mkdir -p ${DRONE_WORKSPACE}
+sudo chown drone:drone ${DRONE_WORKSPACE}
 cd ${DRONE_WORKSPACE}
```

Comment:
looks good. The only request I have is that we should check for `CI=true` and only run the chown commands if we know we are in a CI environment. The reason is that I have unit tests configured to run the tests locally, on the host machine, in a temp directory.

Reply:
`CI=true` or `CI=drone`?  [The docs](http://docs.drone.io/environment-reference/) indicate the latter.

Reply:
I ran a dummy debug plugin to check, and it's `CI=drone`.  I'll wrap the chown in an if statement for that.

Reply:
thanks! instead of looking for `CI=drone` do you think we could check if the `CI` variable is set instead? That way if we ever change the string name (for whatever reason) we don't have to worry about this check breaking.

Reply:
Good thinking.  Just took care of that in fc577cac4bc88b32454c548d88e89b5c379ef0c1.